ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-ssh-client-source-interfaces.j2

```diff
@@ -0,0 +1,20 @@
+{% if ip_ssh_client_source_interfaces is arista.avd.defined %}
+
+## IP ssh Client Source Interfaces
+
+### IP ssh Client Source Interfaces
+
+| VRF | Source Interface Name |
+| --- | --------------- |
+{%     for ip_ssh_client_source_interface in ip_ssh_client_source_interfaces | arista.avd.natural_sort %}
+{%         if ip_ssh_client_source_interface['name'] is arista.avd.defined %}
+| {{ ip_ssh_client_source_interface['vrf'] | arista.avd.default('default') }} | {{ ip_ssh_client_source_interface['name'] | capitalize }} |
```

Comment:
you should not use square brackets for dict keys. We don't do that elsewhere when the key is static. Only use the square brackets for variable keys.
So change `ip_ssh_client_source_interface['vrf']` to `ip_ssh_client_source_interface.vrf` and the same for `name`. Also above.

Reply:
Sure, I checked ip-http-client-source-interfaces.j2 for reference. So I did this way.